internal/storage/postgreSQL/sensorSQL/mqtt.go

```diff
@@ -0,0 +1,142 @@
+package sensorSQL
```

Comment:
I would name it only `sensor`
```suggestion
package sensor
```